### PR TITLE
add section about draft queries

### PR DESCRIPTION
**Pull request by alexthomasTG**

### CODE REVIEW

File: modules/querying/pages/query-operations.adoc [hunk 1]
```diff
@@ -981,7 +1077,10 @@ Additionally, the `ls` GSQL command lists all created queries and identifies whi
 dropQuery := DROP QUERY ( "*" | ALL | queryName ["," queryName]* )
 ----
 
-To drop a query, run `DROP QUERY query_name` __. The query will be uninstalled (if it has been installed) and removed from the dictionary.  The GSQL language will refuse to drop an installed query if another query is installed which calls query Q.  That is, all calling queries must be dropped before or at the same time that their called subqueries are dropped.
+To drop a query, run `DROP QUERY query_name`.
+The query will be uninstalled (if it has been installed) and removed from the dictionary.  The GSQL language will refuse to drop an installed query if another query is installed which calls query Q.  That is, all calling queries must be dropped before or at the same time that their called subqueries are dropped.
```
Comment (xun-gao-tigergraph) [inline modules/querying/pages/query-operations.adoc]: what's `query Q`?

File: modules/querying/pages/query-operations.adoc [hunk 2]
```diff
@@ -970,9 +1011,64 @@ This statement sets a persistent system parameter. Each version of the TigerGrap
 showQuery := SHOW QUERY queryName
 ----
 
-To show the GSQL text of a query, run `SHOW QUERY query_name` . The `query_name` argument can use `*` or `?` wildcards from Linux globbing, or it can be a regular expression when preceded by  `-r`. See  xref:ddl-and-loading:defining-a-graph-schema.adoc#_show__view_parts_of_the_catalog[SHOW: View Parts of the Catalog]
+To show the GSQL text of a query, run `SHOW QUERY query_name` . The `query_name` argument can use `*` or `?` wildcards from Linux globbing, or it can be a regular expression when preceded by  `-r`.
+See  xref:ddl-and-loading:defining-a-graph-schema.adoc#_show__view_parts_of_the_catalog[SHOW: View Parts of the Catalog]
+
+Additionally, the `ls` GSQL command lists all created queries and identifies which queries have been installed as well as their draft or valid status.
```
Comment (xun-gao-tigergraph) [inline modules/querying/pages/query-operations.adoc]: `ls` command not only shows `Queries` section, but contains `Vertex Types`, `Edge Types`, etc.
and identifiers contains `draft`, `installed v2`, `deprecated`, `blocked`, `disabled`, etc.

File: modules/querying/pages/query-operations.adoc (hunk 1 above)
Comment (dadongwang-tg) [reply modules/querying/pages/query-operations.adoc]: I remember "we can drop a subquery, and the main query (no matter installed or not) will be marked as disabled", @xun-gao-tigergraph please test it out.

Comment (xun-gao-tigergraph) [reply modules/querying/pages/query-operations.adoc]: yes, the installed query is able to be dropped.
I tried it as the following output just now.
```
GSQL > create query A(){print 1;}
Successfully created queries: [A].
GSQL > create query B(){ A();}
Successfully created queries: [B].
GSQL > install query *
Start installing queries, about 1 minute ...
<....>
Query installation finished.
GSQL > drop query A
Successfully dropped queries on the graph 'g': [A].
GSQL > ls
---- Graph g
Vertex Types: 
Edge Types: 

Graphs: 
  - Graph g()
Jobs: 
Queries: 
  - B() (disabled)
```

Comment (alexthomasTG) [reply modules/querying/pages/query-operations.adoc]: Thanks for the example!

**Pull request merged**